CHANGELOG.md

```diff
@@ -43,6 +43,15 @@
 
 # `libp2p` facade crate
 
+# 0.45.1
+
+- Update individual crates.
+    - Update to [`libp2p-dcutr` `v0.3.1`](protocols/dcutr/CHANGELOG.md).
+    - Update to [`libp2p-identify` `v0.36.1`](protocols/identify/CHANGELOG.md).
+    - Update to [`libp2p-kad` `v0.37.1`](protocols/kad/CHANGELOG.md).
+    - Update to [`libp2p-relay` `v0.9.1`](protocols/relay/CHANGELOG.md).
+    - Update to [`libp2p-swarm` `v0.36.1`](swarm/CHANGELOG.md).
```

Comment:
Previous entries don't have whitespace at the front.

```suggestion
- Update to [`libp2p-dcutr` `v0.3.1`](protocols/dcutr/CHANGELOG.md).
- Update to [`libp2p-identify` `v0.36.1`](protocols/identify/CHANGELOG.md).
- Update to [`libp2p-kad` `v0.37.1`](protocols/kad/CHANGELOG.md).
- Update to [`libp2p-relay` `v0.9.1`](protocols/relay/CHANGELOG.md).
- Update to [`libp2p-swarm` `v0.36.1`](swarm/CHANGELOG.md).
```

Reply:
Not sure I am following. Isn't this the same for v0.45.0 below?

Reply:
The diff view doesn't show the whitespace 🙃